docs/api-docs/getting-started/api-status-codes.html

```diff
@@ -0,0 +1,262 @@
+<td>When a particular resource doesn’t exist or couldn’t be found.</td>
+</tr>
+<tr>
+<td><strong>405</strong></td>
+<td><strong>Method Not Allowed</strong></td>
+<td>The resource was found, but doesn’t support the request method. Issued when either a specific method isn’t yet implemented on a resource, or the resource doesn’t support the method at all. For example, a <code>PUT</code> on <code>/orders</code> is invalid, but a <code>PUT</code> on <code>/orders/{_id_}</code> is valid.</td>
+</tr>
+<tr>
+<td><strong>406</strong></td>
+<td><strong>Not Acceptable</strong></td>
+<td>When the client specifies a response content type in the <code>Accept</code> header that is not supported.</td>
+</tr>
+<tr>
+<td><strong>409</strong></td>
+<td><strong>Conflict</strong></td>
+<td>A change requested by the client is being rejected, due to a condition imposed by the server. The exact reasons for this response will vary from one resource to the next. An example might be attempting to delete a category whose deletion would cause products to be orphaned. Additional information about the conflict, and about how to resolve it, might be available in the response's <code>details</code> section.</td>
+</tr>
+<tr>
+<td><strong>413</strong></td>
+<td><strong>Request Entity Too Large</strong></td>
+<td>When the client requests too many objects. For example, the <code>limit</code> parameter exceeded the maximum.</td>
+</tr>
+<tr>
+<td><strong>415</strong></td>
+<td><strong>Unsupported Media Type</strong></td>
+<td>Returned due to issues with the <code>Content-Type</code> header.</td>
+</tr>
+<tr>
+<td><strong>422</strong></td>
+<td><strong>Missing or Invalid Data</strong></td>
+<td>The request cannot be processed either because it omitted required fields or because it contained invalid data. See the response for more details.</td>
+</tr>
+<tr>
+<td><strong>429</strong></td>
+<td><strong>Too Many Requests</strong></td>
+<td>When an OAuth client exceeds the <a href="/api-docs/getting-started/basics/best-practices#best-practices_rate-limits">rate limit</a> for API requests to a store.</td>
+</tr>
+</tbody>
+</table>
+<h2 id="5xx-server-error">5xx Server Error</h2>
+<p>5xx codes are returned for requests that could not be processed due to an internal error with the API or server.</p>
+<table>
+<thead>
+<tr>
+<th>Code</th>
+<th>Text</th>
+<th>Purpose</th>
+</tr>
+</thead>
+<tbody>
+<tr>
+<td><strong>500</strong></td>
+<td><strong>Internal Server Error</strong></td>
+<td>When an error has occurred within the API.</td>
+</tr>
+<tr>
+<td><strong>501</strong></td>
+<td><strong>Not Implemented</strong></td>
+<td>When a request method is sent that is not supported by the API (e.g., <code>TRACE</code>, <code>PATCH</code>).</td>
+</tr>
+<tr>
+<td><strong>503</strong></td>
+<td><strong>Service Unavailable</strong></td>
+<td>When the store is either “Down for Maintenance,” being upgraded to a new version, is suspended due to administrative action/a billing issue.</td>
```

Comment:
<td>When the store is “Down for Maintenance,” being upgraded to a new version, or is suspended due to administrative action or a billing issue.</td>